debian/debathena-pyhesiodfs.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Hesiod automounter for Athena lockers
+Requires=openafs-client.service
+After=local-fs.target network.target openafs-client.service
```

Comment:
`openafs-client.service` has `Before=remote-fs.target umount.target` here. Do we need those? i.e. do we count pyhesiodfs as a "remote FS"?